### DISCUSSION TITLE
Change Price after verification

### DISCUSSION THREAD

**Discussion by nftkk** [hashlips-lab/nft-erc721-collection#253] [General]
Hello
everything for my project is done. now i was too cheap. could i change the price on the same smart contract? how? just change price and open the sale´s spot?

**Comment by liarco** [comment]
Yes, after verification you can change the configuration and run the CLI commands without any problem.

If you make changes in between deployment and verification your verification might fail, the reason why this happens is because some of those configuration values are used as constructor arguments in order to save gas when you open the first sale, but the constructor arguments are required during verification.

**Comment by nftkk** [comment]
how do you mean this? i do the changes in the config and then i have to do the verfication again? or just open for example the whitelist sale an the new price will be there?

the verfication is already done with the wrong amounts.

**Reply by liarco** [reply]
Yes, simply change the value and open the sales using the commands.

No need to verify again, I was just explaining that this might break the verification if you make the changes before verifying... but this was just for future users getting here for the same stuff.
